vinhloc30796/creativecontact-app
# feat!: upgrade to NextJS 15 & React 19

## Description

This PR upgrades the project to NextJS 15 and React 19. It includes several improvements and optimizations that come with the new versions.

## Key Changes

1. Upgraded NextJS to version 15.
2. Upgraded React to version 19.
3. Refactored components and code to be compatible with the new versions.

## Breaking Changes

- The upgrade to NextJS 15 and React 19 may introduce breaking changes due to deprecated features and changes in the new versions.

## Related Issues

- Closes # [Issue number]
- Addresses # [Issue number]

## Testing Instructions

1. Pull the branch and install dependencies.
2. Run the project locally and verify that it starts without errors.
3. Test the main features and ensure they work as expected.

## Additional Notes

- Please review the changes carefully and test thoroughly to ensure compatibility with the new versions.
